mirrord-agent/src/env.rs

```diff
@@ -0,0 +1,150 @@
+use std::{
+    collections::{HashMap, HashSet},
+    path::PathBuf,
+};
+
+use mirrord_protocol::RemoteResult;
+use tokio::io::AsyncReadExt;
+use tracing::trace;
+use wildmatch::WildMatch;
+
+struct EnvFilter {
+    include: Vec<WildMatch>,
+    exclude: Vec<WildMatch>,
+}
+
+impl EnvFilter {
+    fn new(select_env_vars: HashSet<String>, filter_env_vars: HashSet<String>) -> Self {
+        let include = if select_env_vars.is_empty() {
+            vec![WildMatch::new("*")]
```

Comment:
I think default values should be managed in config (or at least in the layer) rather than here

Reply:
This isn't quite a default value, it's more like the check that was there previously that will pass the variable if `select_env_vars.is_empty()`

Reply:
If you pass to cli `-x FOOBAR` the filter value in mirrord-layer the values should be `include = None` and `exclude = Some("FOOBAR")` but here I need the `include` to be `*` to allow the exclude (`"FOOBAR" | "HOME" | "PATH"`) filter to work and get all the other env vars